lib/imagery/list_subgp.c

```diff
@@ -135,3 +137,38 @@ int I_list_subgroup_simple(const struct Ref *ref, FILE * fd)
 {
     return I_list_group_simple(ref, fd);
 }
+
+/*!
+ * \brief Formats map name to fit in a 80 column layout
+ *
+ * Results in a map name in the "<map@mapset>" form.
+ * If necessary truncates relevant part(s) and denotes
+ * with ellipsis, e.g. "<verylongmapname...@mapset>".
+ *
+ * \param[out] buf formatted map name
+ * \param name map name
+ * \param mapset mapset name
+ */
+static void name_fitted(char *buf, const char *name, const char *mapset)
+{
+    char *frmt;
+    char fr[32];
+    int name_length = (int)strlen(name);
+    int mapset_length = (int)strlen(mapset);
+
+    if (name_length + mapset_length + 3 < 75) {
+        frmt = "<%s@%s>";
+    }
+    else if (name_length > 35 && mapset_length > 35) {
+        frmt = "<%.33s...@%.32s...>";
+    }
+    else if (name_length > 35) {
+        sprintf(fr, "<\%%.%ds...@%%s>", 68 - mapset_length);
```

Comment:
Why backslash here? GitHub syntax highlighter thinks it is somehow wrong and marks it red. No backslash should be needed here.

Reply:
Honestly don't remember if it's mistake or deliberate. Will look into it.

Reply:
Looking closer at the code, it’s obvious this was deliberate. It creates a format string, which is used in line 173.

I guess GitHubs syntax highlighting means: Are you sure this is what you wanted?

Reply:
I still think the backslash is not needed. `\` is escape of a special character in C string literal, not an escape of _sprintf_ syntax. The right "escape" is doubling the `%` sign which is what you are doing. Try the following code or [run it online](https://onlinegdb.com/180q-2sjh):

```c
#include <stdio.h>
#include <string.h>

int main()
{
    printf("<%%s@%%.%ds...>\n", 100);
    printf("<\%%s@%%.%ds...>\n", 100);
    printf("%d\n", strcmp("%%s", "\%%s"));
    return 0;
}
```

```
<%s@%.100s...>
<%s@%.100s...>
0
```

Reply:
You're absolutely right, the backslash shouldn't be there. Now back at my machine, I also did some testing. Funny thing is, it works either way. Removed them all with latest push.